src/save.js

```diff
@@ -49,6 +49,7 @@ export default function MapsBlockAppleSave(props) {
 					data-subtitle={marker.subtitle}
 					data-color={marker.color}
 					data-glyph-color={marker.glyphColor}
+					data-glyph-image={marker.glyphImage}
```

Comment:
Have we verified that this change doesn't warrant adding a block deprecation? 
We are modifying the markup generated by the save function so I think we need to handle that. 

In order to test this create a map with the plugin before this update and then try to edit it after this update was applied. My guess is that it will throw an error with the deprecation warning. 

You can learn more about how to handle these here: https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-deprecation.md

Reply:
Thanks for raising this @fabiankaegy, I added a "deprecated" version of the block to handle block deprecation here.